src/hooks/useCanJurorVoteFinalRound.js

```diff
@@ -0,0 +1,10 @@
+import { useCourtConfig } from '../providers/CourtConfig'
+import { useActiveBalanceOfAt } from './useCourtContracts'
+
+export default function useCanJurorVoteFinalRound(account, draftTermId) {
```

Comment:
```suggestion
export default function useCanJurorVoteFinalRound(account, disputeDraftTermId) {
```

I (think I) understand this, but it took a bit to realize we needed the checkpointed balance of a juror because it's based on the dispute's draft term. The original naming is also a bit weird because final rounds are associated with disputes, not draft terms.

Reply:
Yeah, thats correct. 
This makes me realize that we probably don't need a hook for this and we can do it directly on the `DisputeVoting` component where we explain the use of the hook